.jenkins/pipelines/standalone/send-email.Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
```

Comment:
I would not recommend a separate Jenkinsfile/pipeline just for sending emails out.

If you do want to invest time into building a modular workflow, a library would be a better alternative.

For example, a library function would allow you to pass in parameters directly without passing `BUILD_URL` through 3 pipelines just so the email would be able to have the url in the body.

Reply:
Totally agreed. I figured that the library could be created in a future ticket, since there's a lot of redundancy that could be reduced from creating a library.

Reply:
Ok -- to do later then :)